docs: add blog introducing-dbcost (#574)
bytebase/bytebase.com: content/blog/introducing-dbcost.md

---
title: Introducing DB Cost, the simple cloud database pricing sheet for AWS RDS and Google Cloud SQL instance
author: Xiong
published_at: 2022/11/02 14:00
feature_image: /static/blog/introducing-dbcost/background.webp
tags: Announcement

> description: DB Cost is an open source project built by the Bytebase team. We expect it to be a tool for developers to compare different database products before making a final decision, where all cloud database instance performance and cost info are clearly presented.

Reviewer: ```suggestion
description: DB Cost is a tool for developers to compare performance and cost differences among cloud database providers.
```

---

## 💸 What is DB Cost?

As FinOps rises, teams become more sensitive to the cloud resource pricing than ever. Major public cloud vendors do provide pricing for their offerings, however, that pricing information is usually hard to consume. Moreover, there is no straightforward way to compare similar offerings across different cloud providers.

> 3rd party pricing calculator site has emerged to fill such a gap. Yet, we couldn't find a site dedicated to providing database pricing info. Being a database tooling shop ourselves, we'd like to take a shot. So today, we are announcing [DB Cost](https://dbcost.com), a simple pricing calculator and comparison tool for the cloud databases. Major features include:

Reviewer: ```suggestion
3rd party pricing calculator site has emerged to fill such a gap. Yet, we couldn't find a site dedicated to providing database pricing info. Being a database tooling shop ourselves, we'd like to take a shot. So today, we are announcing [DB Cost (https://dbcost.com)](https://dbcost.com), a simple pricing calculator and comparison tool for the cloud databases. Major features include:
```
